### ISSUE TITLE
Assess suitability of an XML-based document encoding for OGC documents that can be editing with our current means and publish to multiple templates

### ISSUE THREAD

**Issue by ogcscotts** [ogcscotts/TC-Meeting-topics#73]

**Comment by ogcscotts** [comment]
From TC Closing Plenary:
     This sounds like a content management system; question of scale; the answer to scale justifies how much investment in a content management system is justified
                
                Anything that 'steps us away' from linkable HTML documents...should be very conservatively evaluated
                
                The document as a model; pull from the model
                Database somewhere in the model
                
                What are the tradeoffs between the ease of writing things down; the ease of finding what you need; and the ease of the staff in performing the consortium standards publishing functions
                
                Start with XML sounds great, but have to answer the question:  how are we actually going to actually generate the XML
                
                Discussion about the CalConnect toolchain; full tool chain being offerred for us "for free"; but potentially locks us into a toolchain
                
                Maybe instead of one system, what's the easiest way to deal with multiple systems

**Comment by PeterParslow** [comment]
A dozen years or so ago, this seemed to be the leading contender: https://docbook.org/. It's still being developed, but I don't know if it has been over taken.

It's an open standard (in the sense of 'community developed') specifically designed for managing content & publishing to multiple outputs. Old, but therefore with plenty of mature tools.

That said, the project I was on (for UKHO) which looked at it chose to go with a prioprietary Adobe (Framemaker) system instead - the final output was more 'printer friendly', but rather hard to drive!

**Comment by desruisseaux** [comment]
I have used docbook some years ago for technical documentation, but finally abandoned it. The reason is that the Docbook language looks like HTML but with completely different XML elements (e.g. `<para>` instead of `<p>`). It forces the writer to learn an HTML-like language instead of the much more widespread HTML. It was still justifiable at the time of HTML 4 and previous versions, since Docbook elements are about semantic and HTML was more about rendering. But HTML 5 provides more semantic elements than at the time Docbook was created, and more rendering aspects can be externalized to CSS files. So today I write my documentation directly in HTML 5, trying to limit myself to HTML element that describes semantic and leave rendering aspects apart.

**Comment by PeterParslow** [comment]
Fair enough - I did say the experience this input was based on was rather dated.

Problem is that HTML5 (like previous versions) is not (necessarily) well formed XML, but XHTML5 is not well supported in browsers. If that's not a problem, then "Start with XML sounds great, but have to answer the question:  how are we actually going to actually generate the XML" becomes "Starting with XML is not necessary, if HTML5 satisfies the foreseeable requirements. Quesstion is, how are we going to generate the HTML5?"
